src/main/java/sx/blah/discord/handle/impl/obj/Guild.java

```diff
@@ -736,6 +739,82 @@ public IEmoji getEmojiByName(String name) {
 	}
 
 	@Override
+	public IWebhook getWebhookByID(String id) {
+		IWebhook webhook = channels.stream()
+				.map(IChannel::getWebhooks)
+				.flatMap(List::stream)
+				.filter(hook -> hook.getID().equalsIgnoreCase(id))
+				.findAny().orElse(null);
+
+		if (webhook == null) {
```

Comment:
What is the purpose of this second loop? Won't it always return null if the stream did?